lldb/docs/conf.py

```diff
@@ -89,9 +89,13 @@
 # The suffix of source filenames.
 source_suffix = {
     ".rst": "restructuredtext",
-    ".md": "markdown",
 }
 
+# Man pages do not use markdown pages, so we don't need to register a markdown
+# parser.
+if not building_man_page:
+    source_suffix[".md"] = "markdown"
+
```

Comment:
I'd move `source_suffix` up to before the previous `if not building_man_page:` so that you can add to it during that if instead.